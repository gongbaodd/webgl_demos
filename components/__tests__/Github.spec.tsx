import React from "react";
import { shallow } from "enzyme";
import GitHub from "../Github";

test("hello", () => {
  const wrapper = shallow(<GitHub />);
  expect(wrapper).toMatchSnapshot();
});
