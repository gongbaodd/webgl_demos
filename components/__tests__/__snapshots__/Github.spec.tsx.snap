// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`hello 1`] = `
<GithubCorner
  ariaLabel="Open Github"
  bannerColor="#151513"
  direction="right"
  href="https://github.com/gongbaodd/webgl_demos"
  octoColor="#fff"
  size={80}
/>
`;
